RCAIDE/Methods/Aerodynamics/Airfoil/compute_airfoil_properties_from_panel_method.py

```diff
@@ -7,19 +7,19 @@
 # ----------------------------------------------------------------------
 #  Imports
 # ----------------------------------------------------------------------
-
-import Legacy.trunk.S as SUAVE
+import RCAIDE
 from Legacy.trunk.S.Core import Data , Units
```

Comment:
These should import from RCAIDE:
from RCAIDE.Core import Data, Units

At some point there will be new Data and Units modules and we shouldn't have to hunt down all the places it breaks.